2024-04-14 15:32:52,226 - INFO - GNAR IS STARTING)
2024-04-14 15:35:15,384 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x0000021AA5B985B0>)
2024-04-14 15:35:15,431 - INFO - DB CONNECTION CLOSED)
2024-04-14 15:36:03,113 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 15:36:03,113 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 15:45:14,732 - INFO - GNAR IS STARTING)
2024-04-14 15:45:16,689 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x00000226C6490160>)
2024-04-14 15:45:16,705 - INFO - DB CONNECTION CLOSED)
2024-04-14 15:45:16,736 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x00000226C68DF100>)
2024-04-14 15:45:16,752 - INFO - DB CONNECTION CLOSED)
2024-04-14 15:48:36,689 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x00000226C68DF0D0>)
2024-04-14 15:48:36,705 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:02:41,233 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 16:02:41,233 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 16:08:16,952 - INFO - GNAR IS STARTING)
2024-04-14 16:08:19,838 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x000001220AA5A670>)
2024-04-14 16:08:19,916 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:16:58,803 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x000001220AA5A8E0>)
2024-04-14 16:16:58,865 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:16:58,881 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x000001220AA5A9A0>)
2024-04-14 16:16:59,101 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:23:42,431 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 16:23:42,431 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 16:40:59,642 - INFO - GNAR IS STARTING)
2024-04-14 16:47:50,983 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x0000029CDCBA9700>)
2024-04-14 16:47:51,014 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:59:27,825 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x0000029CDCBA9910>)
2024-04-14 16:59:27,825 - INFO - DB CONNECTION CLOSED)
2024-04-14 16:59:27,840 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x0000029CDD1C74F0>)
2024-04-14 16:59:27,860 - INFO - DB CONNECTION CLOSED)
2024-04-14 17:04:02,109 - INFO - CONN ESTABLISHED <pypyodbc.Connection object at 0x0000029CDD1C7F70>)
2024-04-14 17:04:02,115 - INFO - DB CONNECTION CLOSED)
2024-04-14 17:04:02,131 - ERROR - INPUT ERROR: local variable 'curs' referenced before assignment 6 IS NOT A VALID SELECTION)
2024-04-14 17:04:36,964 - INFO - GNAR IS EXITING EXIT CODE 0)
2024-04-14 17:04:36,964 - INFO - GNAR IS EXITING EXIT CODE 0)
